src/odemis/driver/ueye.py

```diff
@@ -361,11 +361,13 @@ class SENSORINFO(Structure):
 
 SENSOR_UI1545_M = 0x0028  # SXGA rolling shutter, monochrome, LE model
 SENSOR_UI1240LE_M = 0x0054  # SXGA global shutter, monochrome, single board
+SENSOR_UI527xSE_M = 0x238
```

Comment:
Could you add a comment with a simple description of the camera (including that it's GigE)?